### DISCUSSION TITLE
1.19.2 support

### DISCUSSION THREAD

**Discussion by JustN00ne** [kous500/CurveBuilding#25] [Ideas]
Hey, i would like to ask for a nice feature, 1.19.2 Fabric support. It is very usefull as most modifications support it. I tried porting the mod myself, but i failed, 
![image](https://github.com/user-attachments/assets/329f88ee-8e75-4407-8a44-eeafa9ee21e6)

So, i would like atleast 1 singular version that supports 1.19.2, thanks :)

**Comment by kous500** [comment]
Added fabric 1.19.2 to the list of new version candidates.
